checkLibDeps.py

```diff
@@ -43,7 +44,10 @@ def checkPkg(self, pkg):
 
 
 def main():
-    import argparse
+    try:
+        import argparse
+    except ImportError:
+        import argparse_py26 as argparse
```

Comment:
@iarspider , this does not work in `cmssw-slc6` container
```
cmssw-slc6 -- python -c 'import argparse_py26 as argparse'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named argparse_py26
```

Reply:
@smuzaffar it should work if you run it in cms-bot area (the backported module is a part of this PR: https://github.com/cms-sw/cms-bot/pull/1757/files#diff-b2463dea70353cc2d26e61467bc8cf63af14e43550da15fe9d56ca2ad33cb646)

Reply:
I would suggest to drop `argparse_py26`  use the `archived_argparse` ( e.g see /cvmfs/cms.cern.ch/share/overrides/python/tweak_program_helpers.py ). `/cvmfs/cms.cern.ch/share/overrides/python` is in PYTHONPATH once we have cmsenv.